BUTTON_IDEAS.md

```diff
@@ -53,3 +53,16 @@ When adding ideas, please add them in the right category (or add a new one if yo
 | status  | Button name                    | Description                          | Submitted by                                  |
 | ------- | ------------------------------ | ------------------------------------ | --------------------------------------------- |
 | &#9744; | **Add Rounded Social Buttons** | A social button with rounded corners | [Neeraj Gupta](https://github.com/Neeraj3508) |
+
+## Hover Buttons
+
+| status  | Button name      | Description                                                      | Submitted by                                |
+| ------- | ---------------- | ---------------------------------------------------------------- | ------------------------------------------- |
+| &#9744; | **Sleek Button** | A sleek button whose background and outline change upon hovering | [Preity Priya](https://github.com/preity-p) |
+
+
+## Outline Buttons
+
+| status  | Button name      | Description                                                                       | Submitted by                                |
+| ------- | ---------------- | --------------------------------------------------------------------------------- | ------------------------------------------- |
+| &#9744; | **Trace Button** | An outline button whose border gets traced, starting and ending at the same point | [Preity Priya](https://github.com/preity-p) |
```

Comment:
Please add this button idea under Animated buttons

Reply:
sure, done!